app/src/main/res/drawable/ic_action_star_list.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
_Reposting the comment in a resolvable thread_

>> The icons you added appear much smaller on the device (Pixel 6) I tested with. Please try to match the former dimensions. I find them too small to recognize the details.

> I am seemingly not able to match it 100%, but it should be better now (as seen in the Screenshot's I added)

What blocks you here from matching the size?

Reply:
I think it's because of the switch from handmade Icons to icons provided by Google. I can get them to a difference of a few pixels, but I think the base Vector Image has ever so slightly different properties so I would have to spend a lot of time fine-tuning that.

Reply:
I can hopefully help with that. The icons are still too small to be recognizable very well. I try to find out how to fix it.

Reply:
I made some progress on a [separate branch](https://github.com/EventFahrplan/EventFahrplan/tree/switchToVectorImages_refinement). There are indeed some fine-tuning steps which need time and patience. I will continue there once I have time. :hourglass_flowing_sand:

Reply:
Here is the current state of the icons comparing `master` (d6480389f87df94f499fb7a9bd26f49f808b99e1) with the state of the separate branch (9bfe7aa2307f163584e3334fbb9a665ef9625e80):

# Schedule screen (before / after)
- Icons in the toolbar
- Camera icon on a session view
![20230120_schedule_screen_tablet](https://user-images.githubusercontent.com/144518/213721153-980b6e70-2cb6-416e-a9b2-83c3ec029585.png)

# Favorites screen (before / after)
- Camera icon on a past session view
- Camera icon on a future session view
![20230120_favorites_screen_tablet](https://user-images.githubusercontent.com/144518/213721182-c4880b7e-3a9a-4bfd-88bf-d697f6ca9a6f.png)

These look good to me. Any objections?

Reply:
Nope, looks good to me :)